job_bundles/README.md

```diff
@@ -0,0 +1,315 @@
+Note that the `BlenderSceneFile` and `OutputDir` Job Parameters are both defined with `type: PATH`, so `{{Param.BlenderSceneFile}}` and
+`{{Param.OutputDir}}` will resolve to the [path-mapped location](https://github.com/OpenJobDescription/openjd-specifications/wiki/How-Jobs-Are-Run#path-mapping)
+on the Worker where the file and directory are located.
+After the embedded file has been written
+to disk, the Task's `onRun` command is resolved to `bash <location-of-embeded-file>` and then run.
+
+Please see the Open Job Description specification for additional details on [how Jobs are run](https://github.com/OpenJobDescription/openjd-specifications/wiki/How-Jobs-Are-Run).
+
+Additional samples of Job Templates can be found in all of the Job Bundles in this repository as well as in the 
+[samples provided](https://github.com/OpenJobDescription/openjd-specifications/tree/mainline/samples)
+by Open Job Description. Please see https://github.com/OpenJobDescription/openjd-specifications/wiki
+for more information on Job Templates.
+
+### Elements - Parameter Values
+
+The `parameter_values.yaml`/`parameter_values.json` file in a Job Bundle gives you a place to "bake"
+the values of some of the Job Parameters, and/or [deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html)
+request arguments into the Job Bundle so that the values do not have to be given when submitting a Job using the
+Job Bundle. The Job Bundle submission UI that is created will still allow values for these parameters to be modified.
+
+The format of the file, in YAML, is:
+
+```yaml
+parameterValues:
+- name: <string>
+  value: <integer>, <float>, or <string>
+- name: <string>
+  value: <integer, <float>, or <string>
+... repeating as necessary
+```
+
+Each element of the `parameterValues` list in the file can be one of the following: 
+
+1. A Job Parameter defined in the Job Bundle's Job Template; 
+2. A Job Parameter defined in a Queue Environment on the Queue that you are submitting the Job to;
+3. A special parameter that is passed to the [deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html)
+   when creating a Job with the Job Bundle:
+    * `deadline:priority` - The value must be an integer, and is passed as the `priority` request parameter to the API.
+    * `deadline:targetTaskRunStatus` - Value must be a string, and is passed as the `targetTaskRunStatus` request parameter to the API.
+    * `deadline:maxFailedTasksCount` - Value must be an integer, and is passed as the `maxFailedTasksCount` request parameter to the API.
+    * `deadline:maxRetriesPerTask` - Value must be an integer, and is passed as the `maxRetriesPerTask` request parameter to the API.
+
+While a standalone Job Template is viewed as a template from which Jobs can be created, a Job Bundle can be viewed as both
```

Comment:
A standalone template could be viewed as either one as well. E.g. a template that takes no parameters will define one specific job. I'd suggest to remove the contrast here, and just talk about the job bundle this way. "A job bundle can function as either a template from which to create jobs, or as a single specific job."

Reply:
See https://github.com/aws-deadline/deadline-cloud-samples/pull/13#discussion_r1663002017. I felt the same way. Even a job bundle with parameter values for all parameters can be used like a template by providing overridden values.